File: python/Tayangan.py
class Tayangan:
    def __init__(self, id, judul, durasi, jamTayang, hargaTiket):
        self.id = id
        self.judul = judul
        self.durasi = durasi
        self.jamTayang = jamTayang
        self.hargaTiket = hargaTiket

    # getter
    def get_id(self):
        return self.id
    def get_judul(self):
        return self.judul
    def get_durasi(self):
        return self.durasi
    def get_jamTayang(self):
        return self.jamTayang
    def get_hargaTiket(self):
        return self.hargaTiket

    # setter
    def set_id(self, id):
        self.id = id
    def set_judul(self, judul):
        self.judul = judul
    def set_durasi(self, durasi):
        self.durasi = durasi
    def set_jamTayang(self, jamTayang):
        self.jamTayang = jamTayang
    def set_hargaTiket(self, hargaTiket):
        self.hargaTiket = hargaTiket

File: php/Tayangan.php
<?php
class Tayangan{
    // atribut
    private $id;
    private $gambar;
    private $judul;
    private $durasi;
    private $jamTayang;
    private $hargaTiket;

    // konstruktor
    public function __construct($id, $gambar, $judul, $durasi, $jamTayang, $hargaTiket){
        $this->id = $id;
        $this->gambar = $gambar;
        $this->judul = $judul;
        $this->durasi = $durasi;
        $this->jamTayang = $jamTayang;
        $this->hargaTiket = $hargaTiket;
    }

    // getter
    public function getId(){
        return $this->id;
    }
    public function getGambar(){
        return $this->gambar;
    }
    public function getJudul(){
        return $this->judul;
    }
    public function getDurasi(){
        return $this->durasi;
    }
    public function getJamTayang(){
        return $this->jamTayang;
    }
    public function getHargaTiket(){
        return $this->hargaTiket;
    }

    // setter
    public function setId($id){
        $this->id = $id;
    }
    public function setGambar($gambar){
        $this->gambar = $gambar;
    }
    public function setJudul($judul){
        $this->judul = $judul;
    }
    public function setDurasi($durasi){
        $this->durasi = $durasi;
    }
    public function setJamTayang($jamTayang){
        $this->jamTayang = $jamTayang;
    }
    public function setHargaTiket($hargaTiket){
        $this->hargaTiket = $hargaTiket;
    }
}

?>